packages/@core.keystone/cache.js

```diff
@@ -131,5 +128,6 @@ const initCache = (keystone, cache) => {
 
 module.exports = {
     KeystoneCacheMiddleware,
+    initCache,
```

Comment:
Looks like this export should be removed after this refactoring 5b969cb5ea9d1c4b3e4d2a1e2af33d1b85218fa4